tests/proposals/ccip-test-proposal-001.clar

```diff
@@ -0,0 +1,14 @@
+;; Title: Test Proposal
+;; Author: City Coins
```

Comment:
The other contracts are structured a little different for metadata, can we match the format there? e.g. for ccd001:

- Title
- Version
- Synopsis
- Description

We *could* add an author field to each one with the value "CityCoins" but wasn't sure that'd be helpful.

Reply:
My bad - copy paste error - happy to conform to your conventions.